tests/integration/file/test_file_client.py

```diff
@@ -203,3 +205,45 @@ def test__back_off_retry__works(self, client: FileClient, test_file):
 
         for i in range(20):
             rename_file(client=client, file_id=file_id, name=f"{PREFIX}{i}.abc")
+
+    def test__query_files_linq__filter_by_name_succeeds(
+        self, client: FileClient, test_file, random_filename_extension: str
+    ):
+        file_id = test_file(file_name=random_filename_extension)
+
+        query_request = FileLinqQueryRequest(
+            filter=f'name == "{random_filename_extension}"'
+        )
+        response = client.query_files_linq(query=query_request)
```

Comment:
Use the entire API surface in your test to make sure we are properly building all fields of the request.